### PR TITLE
 Move usr/bin declaration to top of file 

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: Sonia Singla <soniasingla.1812@gmail.com>

### CODE REVIEW

File: pkg/webhook/cspc_test.go [hunk 1]
```diff
@@ -740,10 +740,6 @@ func TestValidateCSPCUpdateRequest(t *testing.T) {
 
 				// Fetch blockdevice details and update
 				for _, bd := range getfakeBDs(test.updatedHostName, test.blockdevicePrefix, 7) {
-					_, err = f.wh.clientset.OpenebsV1alpha1().
-						BlockDevices(bd.Namespace).
-						Update(context.TODO(), bd, metav1.UpdateOptions{})
-				}
```
Comment (mittachaitu) [inline pkg/webhook/cspc_test.go:746]: any reason for removing this code snippet? It is required to test the following [case](https://github.com/openebs/cstor-operators/blob/b82dfa4ef36e6141c1d5dd00491d7344f054c521/pkg/webhook/cspc_test.go#L631).

Comment (soniasingla) [reply pkg/webhook/cspc_test.go]: sorry removed unintentionally

File: ci/sanity/install.sh [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
 # Copyright © 2020 The OpenEBS Authors
```
Comment (mittachaitu) [inline ci/sanity/install.sh:2]: Let's maintain a line space after mentioning env
```suggestion
#!/usr/bin/env bash

# Copyright © 2020 The OpenEBS Authors
```

File: ci/sanity/sanity.sh [hunk 3]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
 # Copyright © 2020 The OpenEBS Authors
```
Comment (mittachaitu) [inline ci/sanity/sanity.sh:2]: Same as above comments let's maintain line space after env
```suggestion
#!/usr/bin/env bash

# Copyright © 2020 The OpenEBS Authors
```

**Review by sonasingh46 — APPROVED** [review]

**Pull request merged**